packages/contracts/src/PoolMetadataAuthorization.sol

```diff
@@ -0,0 +1,36 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.7.0 <0.9.0;
+
+import "@balancer-labs/v2-interfaces/contracts/vault/IAuthorizer.sol";
+import "@balancer-labs/v2-solidity-utils/contracts/helpers/Authentication.sol";
```

Comment:
`#would` use only imports from `balancer-v2-monorepo/pkg`. Importing from @balancer-labs will require installing npm packages and linking them through remappings which we proved at the beginning of the project is super error prone